synapse/models/person.py

```diff
@@ -60,12 +115,10 @@ def getBaseModels():
 
                 ('ps:contact', {'subof': 'guid', 'doc': 'A GUID for a contact info record'}),
 
-                ('ps:hasuser', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|user,inet:user'}),
-                ('ps:hashost', {'subof': 'comp', 'fields': 'person=ps:person,host=it:host'}),
-                ('ps:hasalias', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|alias,ps:name'}),
-                ('ps:hasphone', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|phone,tel:phone'}),
-                ('ps:hasemail', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|email,inet:email'}),
-                ('ps:haswebacct', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|web:acct,inet:web:acct'}),
+                ('ps:has', {
+                    'subof': 'xref',
+                    'source': 'person,ps:person',
+                    'doc': 'A person that has a thing. FIXME reword'}),
```

Comment:
Suggested: "A person owns, controls, or has exclusive use of an object or resource, potentially during a specific period of time."

Reply:
fixed